packages/editor/src/components/local-autosave-monitor/index.js

```diff
@@ -140,7 +143,7 @@ function useAutosaveNotice() {
 				},
 			],
 		} );
-	}, [ postId ] );
+	}, [ postId, remoteAutosave ] );
```

Comment:
This means, the function is reexecuted each time `removeAutosave` changes which can happen on each autosave I guess and  not only initial rendering, is this expected?

Reply:
No, indeed, not expected. I think I left this here in my frustration with the E2E tests, when for some reason `[ postId, hasFetchedAutosave ]` didn't work. I can circle back.